src/Client/V2/Client.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Lalamove\Client\V2;
+
+use Lalamove\Resources\V2\DriversResource;
+use Lalamove\Resources\V2\OrdersResource;
+use Lalamove\Resources\V2\QuotationsResource;
+
+class Client
+{
+    protected $settings;
```

Comment:
Because we're moving to PHP >= 7.4 we can introduce typed properties:

https://www.php.net/manual/en/language.types.declarations.php

Please update all class properties to be typed; e.g:

```php
protected $settings; // before

protected Settings $settings; // after
```